src/content/self-hosted/install/releases.mdx

```diff
@@ -8,11 +8,41 @@ id: releases
 ## 1.7.0
 TBD
 
+### Removal Notice
+- Kubernetes [1.21](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md) is no longer supported.
+
 ### Features
 
 - Support for the annotation [dev.okteto.com/endpoints](cloud/ssl.mdx#customizing-the-endpoints-shown-in-the-okteto-ui) to customize the endpoints shown in the Okteto UI.
 - Support for configuring the [service account](self-hosted/administration/configuration.mdx#buildkit) of buildkit.
 - Support for [token-based authentication](self-hosted/administration/configuration.mdx#token) to simplify the installation of Okteto Self-Hosted.
+- [Personal Access Tokens](cloud/personal-access-tokens.mdx) now have expiration date. They expire after 180 days of their creation. For existing tokens, the expiration date is set to 180 days from when your instance is upgraded to this version
+- Update the Okteto CLI version to [2.14.1](https://github.com/okteto/okteto/releases/tag/2.14.1)
+- Allow to delete external resources individually from the UI
+- Default installation of Okteto doesn't require an external storage for the registry configured. It uses filesystem storage by default
+- Default installation of Okteto doesn't require a PVC for [BuildKit](self-hosted/administration/configuration.mdx#token). It uses [filesystem storage](self-hosted/administration/configuration.mdx#filesystem) by default
+
+### Improvements
+
+- When `buildkit.ingress.enabled` is true, BuildKit is exposed in the `buildkit.service.port` instead of port 1234. This makes easier to use other ingress controller.
+- Optimized redeploy dialog experience. Infomation loads faster in the dialog, specially when the source is GitHub
+- Improved confirmation dialogs messages
+- When your Okteto instance doesn't have an auth provider configured, the login screen shows a message with a link to the documentation about how to configure it
+- Upgrade ingress-nginx to [4.3.0](https://github.com/kubernetes/ingress-nginx/releases/tag/helm-chart-4.3.0)
+- Pod Security Policies are not installed if they are not available in the cluster
+- Moved `cloud.provider` helm chart value to [`registry.storage.provider`](self-hosted/administration/configuration.mdx#storage). We keep backward compatibility with values configured under `cloud.provider`
+- When using external storage for the registry, the secret with the credentials can be configured now in [`registry.secret`](self-hosted/administration/configuration.mdx#registry). We keep backward compatibility with `cloud.secret` helm value
```

Comment:
I'm confused on how this is different from the one above.

Reply:
One is regarding `cloud.provider` and the other is `cloud.secret` they are related but not the same.

just a proposal :

```suggestion
- When using external storage for the registry,  helm chart value `cloud.provider` has been moved to  [`registry.storage.provider`](self-hosted/administration/configuration.mdx#storage). We keep backward compatibility with values configured under `cloud.provider`
- The secret with the credentials for external storage provider for the registry should be configured now in [`registry.secret`](self-hosted/administration/configuration.mdx#registry). We keep backward compatibility with `cloud.secret`
```

Reply:
ah I see, thanks!